Library/Formula/libxc21.rb

```diff
@@ -0,0 +1,52 @@
+class Libxc21 < Formula
+  desc "Library of exchange and correlation functionals for codes"
+  homepage "http://www.tddft.org/programs/octopus/wiki/index.php/Libxc"
+  url "http://www.tddft.org/programs/octopus/down.php?file=libxc/libxc-2.1.0.tar.gz"
+  sha256 "481fcd811d7f5e99ceab2596be09e422a21e9b03437ca607b9c03ffc42050d29"
+  revision 1
+
+  bottle do
+    cellar :any
+    sha256 "21c14d74f3b61c2a90518208500aee11fc18d0f0449cb76b0d3d30a0e8ec0fe9" => :yosemite
+    sha256 "c6d5ac5639158a1c0c7e66abd31bdfe5fa3d036c702dc64d75d3d4a625897e4a" => :mavericks
+    sha256 "63b05a1fb40be747e31c0d7addf8d424df06d42dd028a907e92ea738d1b0e684" => :mountain_lion
+  end
```

Comment:
please delete this block; our test bot will compute these hashes after building bottles